src/test/java/org/jboss/xavier/analytics/rules/workload/inventory/WorkloadInventoryReportTest.java

```diff
@@ -545,7 +545,7 @@ public void testFlagsCentOS() throws ParseException {
         Assert.assertTrue(targets.contains("RHV"));
         Assert.assertTrue(targets.contains("Convert2RHEL"));
         // Complexity
-        Assert.assertEquals(WorkloadInventoryReportModel.COMPLEXITY_HARD,workloadInventoryReportModel.getComplexity());
+        //Assert.assertEquals(WorkloadInventoryReportModel.COMPLEXITY_HARD,workloadInventoryReportModel.getComplexity());
```

Comment:
Same as the above comment

Reply:
This check is reintroduced with a different value in MIGENG-337

Reply:
Great, thanks @m-brophy